### DISCUSSION TITLE
Can u block Ports in SagerNet?

### DISCUSSION THREAD

**Discussion by wenjucktes01** [SagerNet/SagerNet#498] [Q&A]
Can u block Ports in SagerNet?

**Comment by iKirby** [comment]
You can use routes to block ports. To block things matching the route, set outbound to Block in Route Settings.

**Reply by wenjucktes01** [reply]
For the whole port, i just let all free and fill the port brackets?

**Reply by iKirby** [reply]
You can use `0.0.0.0/0` and `::/0` to match all IP addresses.

**Reply by wenjucktes01** [reply]
Just to be sure, the Port I would like to block I have to write in the Port bracket? And whats the source port? And is it possible to create a more specific dokumentation?

**Reply by malikshi** [reply]
Read this
[Routing doc](https://www.v2fly.org/config/routing.html#%E9%A2%84%E5%AE%9A%E4%B9%89%E5%9F%9F%E5%90%8D%E5%88%97%E8%A1%A8)

**Reply by wenjucktes01** [reply]
Thanks for the Dokument but it did not awnswerd the Port question.

**Reply by wenjucktes01** [reply]
Is it possible to make an Guide wich explains the Route function in the App? I am sorry but I didn't get the things explainded there because I did not know anthing about programing and i think there are more that didnt get it because thes also do not know anything about programming.

**Reply by malikshi** [reply]
> Read this [Routing doc](https://www.v2fly.org/config/routing.html#%E9%A2%84%E5%AE%9A%E4%B9%89%E5%9F%9F%E5%90%8D%E5%88%97%E8%A1%A8)

Already explained there. You're not try anything from July until September?

**Reply by wenjucktes01** [reply]
I had already awnsered in July but nobody anwserd. It seems you do not understand me.

**Reply by wenjucktes01** [reply]
I think i explain it with an Screenshot. Where do i have to put all correctly
![Screenshot_20220927_111348_SagerNet](https://user-images.githubusercontent.com/107936474/192489755-3384c944-9fde-4c30-8795-9560b7ed63b8.jpg)
